ansible_collections/arista/avd/roles/eos_designs/docs/tables/l3-edge.md

```diff
@@ -317,4 +344,83 @@
           # Custom structured config for interfaces
           # Note! The content of this dictionary is _not_ validated by the schema, since it can be either ethernet_interfaces or port_channel_interfaces.
           structured_config: <dict>
+      l3_interfaces_profiles:
+
+          # L3 interface profile name. Any variable supported under `l3_interfaces` can be inherited from a profile.
+        - name: <str; required; unique>
```

Comment:
For the other profiles we often use `profile:` here, since it makes it simpler to implement the merging in code :)

Reply:
This is the name of the profile under `l3_interfaces_profiles`. Using `profile` as primary key is not consistent with the `p2p_links` model but I see we do it indeed in the other (svi, port profile, ...) so let me do it well for this new model!